hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/BucketEndpoint.java

```diff
@@ -327,6 +352,9 @@ public Response listMultipartUploads(
       AUDIT.logReadFailure(
           buildAuditMessageForFailure(s3GAction, getAuditParameters(), ex));
       throw ex;
+    } finally {
+      getLatencyMetrics().addListMultipartUploadsLatencyNs(
+          Time.monotonicNowNanos() - start);
```

Comment:
Doesn't mixing latency for both successful and failed operations distort the picture?

Reply:
@adoroszlai do you recommend I revert back to the initial commit where we only capture the latency once the operation is successful?

Reply:
No, I recommend having two separate latency metrics, one for successful ops, another one for failed ops.